xwayland/xwm.c

```diff
@@ -632,6 +633,22 @@ static void xwm_handle_configure_notify(struct wlr_xwm *xwm,
 	xsurface->y = ev->y;
 	xsurface->width = ev->width;
 	xsurface->height = ev->height;
+
+	struct wlr_xwayland_surface_configure_event *wlr_event =
+		calloc(1, sizeof(struct wlr_xwayland_surface_configure_event));
+	if (wlr_event == NULL) {
+		return;
+	}
+
+	wlr_event->surface = xsurface;
+	wlr_event->x = ev->x;
+	wlr_event->y = ev->y;
+	wlr_event->width = ev->width;
+	wlr_event->height = ev->height;
+
+	wl_signal_emit(&xsurface->events.request_configure, wlr_event);
```

Comment:
You added a new configure_notify but you use request_configure?

Reply:
thx. Right, I had simply copy pasted the code and seems I missed that rather important thing.